bedrock/privacy/views.py

```diff
@@ -69,6 +69,8 @@ def get_legal_doc(self):
     template_name="privacy/notices/subscription-services.html", legal_doc_name="subscription_services_privacy_notice"
 )
 
+mozilla_accounts = PrivacyDocView.as_view(template_name="privacy/notices/mozilla-accounts.html", legal_doc_name="Mozilla_Accounts_Privacy_Notice")
```

Comment:
The doc name has been updated to all lower-case for consistency with all of the other docs.

Reply:
Thanks, updated